jsonschema-hyperschema.xml

```diff
@@ -334,6 +334,16 @@
                 </section>
             </section>
 
+            <section title="deprecated">
+                <t>
+                    If it has a value of boolean true, this keyword indicates that the value of the instance is deprecated, and intended for reverse compatability or internal use only, and may become missing in the future.
```

Comment:
So how does this work?

In a root schema with a "self" link is means that the resource itself is deprecated?  But it's still there currently?

In a root schema without a "self" link it means....?

In a property subschema (in any of the property keywords) it means that all matching properties are deprecated but will still be present?

In a schema applying to a single position in an array/tuple ("items" is an array) it means...?

In a schema applying to multiple array positions ("items" as one schema, or "additionalItems") it means...?

In the schema for "schema" or "targetSchema" it means...?

This concept is not ready for a PR.  There are far too many cases that need discussing and a coherent view of hypermedia resources that need to be developed before we can just toss this in.